src/components/form/submission.vue

```diff
@@ -21,7 +21,7 @@ except according to the terms contained in the LICENSE file.
 </template>
 
 <script setup>
-import { defineOptions, defineAsyncComponent, watchEffect, computed } from 'vue';
```

Comment:
The Vite build included a warning about importing `defineOptions`. The Vue build handles it specially, so it doesn't need to be imported.